src/com/sap/piper/TemporaryCredentialsUtils.groovy

```diff
@@ -0,0 +1,60 @@
+package com.sap.piper
+
+class TemporaryCredentialsUtils implements Serializable {
+
+    private static Script script
+
+    TemporaryCredentialsUtils(Script script) {
+        this.script = script
+    }
+
+    void writeCredentials(List credentialItems, String credentialsDirectory, String credentialsFileName) {
```

Comment:
I would have expected a method that takes a closure similar to what the step did taking care of writing and deleting (not placing that burden on the caller). Do I miss some reason why this does not work?

Reply:
No you are right. I have addressed that and added a method with the latest commit.